lib/parseFile.js

```diff
@@ -103,6 +103,9 @@ module.exports = async function parseFile(file, options = {}) {
               }
 
               property.type = [ 'URI', 'QName', 'ID' ].includes(type) ? 'String' : type;
+            } else if (property.name === 'id') {
```

Comment:
@barmac Can you add a test case for that one?

Reply:
Sure.